cmd/cosi-driver-sample/main.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 The Kubernetes Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+
+	"k8s.io/klog/v2"
+
+	"sigs.k8s.io/cosi-driver-sample/internal/provisionerserver"
+	"sigs.k8s.io/cosi-driver-sample/pkg/cosidriver"
+)
+
+const (
+	driverName = "cosi-driver-sample.sigs.k8s.io"
+)
+
+func main() {
+	klog.InitFlags(nil)
+
+	endpoint := flag.String("endpoint", cosidriver.DefaultEndpoint, "endpoint for the GRPC server")
+	flag.Parse()
+
+	defer klog.Flush()
```

Comment:
We do this in the Scality driver, but I want to keep this one as simple as possible, not adding the cognitive overhead of knowing Cobra and its APIs etc.
Agreed, using Cobra or whichever CLI framework one prefers for a 'production' driver is likely a good idea, but I don't think it has a place in this codebase. Let's keep this simple.

Reply:
I think it would be good to keep a standard shape/style for the cmd.go/main.go files. The basic cobra logic is pretty minimal, and the change-over should be relatively quick.

Reply:
Can be done as a future improvement, if desired.

Reply:
SGTM, will open related issue when this merges